stock_release_channel/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    recompute_channel_on_pickings_at_release = fields.Boolean(
+        help="When releasing a transfer, recompute channel on pickings",
```

Comment:
I'm confused, you favor help over string?

Reply:
For pre-commit, nils
```************* Module stock_release_channel.models.res_company
stock_release_channel/models/res_company.py:10: [W8113(attribute-string-redundant), ResCompany] The attribute string is redundant. String parameter equal to name of variable

Check for Odoo modules using pylint......................................Failed
- hook id: pylint_odoo
- exit code: 4

************* Module stock_release_channel.models.res_company
stock_release_channel/models/res_company.py:10: [W8113(attribute-string-redundant), ResCompany] The attribute string is redundant. String parameter equal to name of variable

Reply:
On xml with `group` tag, it will render the string equal to the name of variable